redhat-developer/web-terminal-tooling
# Update tooling for OCP 4.15; update base images

This PR updates the tooling used in the web terminal operator tooling container to their latest versions available on mirror.openshift.com and github releases for OCP 4.15, as well as updates the UBI8-minimal image used. 

# Testing
I've pushed the web terminal tooling container built from this PR to `quay.io/aobuchow/wto-tooling:1.10` for convenience. 

1. Launch an OpenShift 4.15 cluster
2. Install the web-terminal operator
3. Create a web terminal and click the link to specify a custom image, and provide `quay.io/aobuchow/wto-tooling:1.10`. Alternatively, create a web terminal with the default image and run `wtoctl set image quay.io/aobuchow/wto-tooling:1.10` in the web terminal console
4. Verify the web terminal starts up successfully. The provided binaries should be updated to the versions specified in this PR. For example, running `oc version` should indicate version 4.15.
